.github/workflows/release-next-version.yml

```diff
@@ -0,0 +1,91 @@
+name: CI
+
+on:
+  push:
+  schedule:
+    - cron: '0 0 * * MON' # Every Monday
```

Comment:
I believe the [`schedule` trigger](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule) runs only on the default branch. I'm thinking we may need a higher level workflow to trigger these builds on each branch.

Should we solve that as part of this PR or go ahead and merge this and work on that separately? If separately, perhaps remove the schedule and only include the manual trigger here?

I personally think the manual trigger adds value on other branches even without a scheduled run!

Reply:
At the moment the only way to run a scheduled workflow on the 5.7.x branch is to add another workflow, for example `release-next-version-5.7.x.yml`, to the main branch. This isn't ideal since the CI code won't be tied to the source code, they'll be on different branches.
The `release-next-version-5.7.x.yml` would look like this:
```
# ...
jobs:
  # ...
  release:
    name: Release next version
    needs: [check_release_due]
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
        with:
          ref: 5.7.x
# ...
```
I would like to hold off on adding this until we have a complete release pipeline, in case GitHub has added some new functionality by then.

The relevant GitHub Actions issue is [here](https://github.community/t/scheduled-builds-of-non-default-branch/16306).

Reply:
Thanks! I agree, let's hold off. Just wanted to note it for future me/you/anyone else.